plugin/adsk/plugin/exportTranslator.cpp

```diff
@@ -0,0 +1,211 @@
+//
+// Copyright 2016 Pixar
+// Copyright 2019 Autodesk
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+#include "exportTranslator.h"
+
+#include <mayaUsd/fileio/jobs/jobArgs.h>
+#include <mayaUsd/fileio/shading/shadingModeRegistry.h>
+#include <mayaUsd/fileio/jobs/writeJob.h>
+#include <mayaUsd/fileio/utils/writeUtil.h>
+
+#include <maya/MFileObject.h>
+#include <maya/MGlobal.h>
+#include <maya/MSelectionList.h>
+#include <maya/MString.h>
+
+#include <string>
+
+PXR_NAMESPACE_USING_DIRECTIVE
```

Comment:
not super familiar with this but why are we using PXR_NAMESPACE_USING_DIRECTIVE here while the importer uses PXR_NAMESPACE_OPEN_SCOPE ?

Reply:
I believe the importer is incorrect.  So far both import and export are basically copied from the Pixar plugin and I think the source for the ADSK importer was copy/pasted from Pixar and then edited a bit but the Pixar namespace wasn't changed.  PXR_NAMESPACE_OPEN_SCOPE puts the class into the pxr namespace but the ADSK plugin has it’s own.  I also copy/pasted a lot of what the Pixar plugin has in its export code, but when I went with the MayaUSD namespace for the exporter I had to either add a “pxr::” to the start of some of the functions or just use that macro to add a “using pxr”.